### PR TITLE
Desktop1

### PR DESCRIPTION
**Pull request by Richez003**
desktop page submission

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -455,8 +461,344 @@ ul.card-item {
   line-height: 32px;
   text-align: center;
   color: #42526e;
+  font-family: 'inter', sans-serif !important;
 }
 
-.work-section {
-  display: grid;
+@media screen and (max-width: 720px) {
```
Comment (rustam-tolipov) [inline style.css:461]: **You made a great effort on this project! I love it!**
_However, some places need to be fixed for the next step. If you are with me let's get started! 😄_

- [x] You used media queries to make the website responsive. It's perfect! 
But, when it comes to width range it should be different. 
The value for the min-width property should be in the range of 768px or wider. Please follow this requirement!

File: index.html [hunk 2]
```diff
@@ -245,35 +269,57 @@ <h1>About Me</h1>
...
-                   
-                    <input type="text" maxlength="30" placeholder="Full Name" required>
-            
-                
-                    <input type="email" name="email" placeholder="Enter address" required>
-            
+                   <div class="form-style">
+                       <div class="input">
+                    <input type="text" maxlength="30" placeholder="First Name" required>
+                    <input type="text" maxlength="30" placeholder="Last Name" required>
```
Comment (rustam-tolipov) [inline index.html:282]: - [x] In this place, I noticed that. Your `Last Name` input is slightly smaller than the original design. As you can see in this image. Please make it equal width with other inputs. Such as, `Enter address`, `Enter text here`...

Screen Shot 2022-04-02 at 4 31 05 PM

File: index.html [hunk 3]
```diff
@@ -245,35 +269,57 @@ <h1>About Me</h1>
...
+                <div><a href="https://angel.co/jobs?ref=onboarding"><img src="./images/angellist.png" alt="angellist"></a></div>
+          </div>
+                <a class="last-image" href="#"><img src="./images/illustration-contact-form.png" alt="illustrator"></a>     
+        </section>
+        <div>
+        <p class="lp">Richemonday@gmail.com</p>
+        </div>              
+            <div>  
+                    <a class="image5" href="#"><img src="./images/illustration-contact-form.png" alt="illustrator"></a>    
+                <!-- <a href="" > <img src="./images/Frame@2x.png" alt="" class="last-frame"></a> -->
```
Comment (rustam-tolipov) [inline index.html]: - [x] To make your code clean...
Please, clear your project from unused commented codes!

File: style.css [hunk 4]
```diff
@@ -231,7 +230,8 @@ ul.card-item {
 
 .items ul {
   display: flex;
-  justify-content: center;
+
+  /* justify-content: center; */
```
Comment (rustam-tolipov) [inline style.css]: - [x] To make your code clean...
Please, clear your project from unused commented codes!

File: style.css [hunk 5]
```diff
@@ -251,8 +251,9 @@ ul.card-item {
 }
 
 .card-lists h3 {
-  text-align: center;
-  font-size: 30px;
+  /* text-align: center;
+  font-size: 30px; */
```
Comment (rustam-tolipov) [inline style.css]: - [x] To make your code clean...
Please, clear your project from unused commented codes!

File: style.css [hunk 6]
```diff
@@ -282,10 +283,11 @@ ul.card-item {
...
 }
 
-.pic3 {
+.pic4 {
   text-align: end;
-  margin: -3%;
-  position: relative;
+  margin-left: 85%;
+
+  /* position: relative; */
```
Comment (rustam-tolipov) [inline style.css]: - [x] To make your code clean...
Please, clear your project from unused commented codes!

File: style.css [hunk 7]
```diff
@@ -455,8 +461,344 @@ ul.card-item {
...
+    height: 48px;
+    padding-left: 550px;
+    margin-top: -115px;
+    font-family: 'inter', sans-serif !important;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+
+    /* or 130% */
```
Comment (rustam-tolipov) [inline style.css]: _I think it is the necessary comment in your code! However, please live more explained comments to make it beautiful to understand and see! 😉_

- [x] To make your code clean...
Please, clear your project from unused commented codes!

File: style.css [hunk 8]
```diff
@@ -455,8 +461,344 @@ ul.card-item {
...
+    font-size: 30px;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+    color: #172b4d;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+
+    /* margin: 24px 0; */
```
Comment (rustam-tolipov) [inline style.css]: - [ ] To make your code clean...
Please, clear your project from unused commented codes!

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello TEAM, 

Good job so far!

### HIGHLIGHTS!

- No linters errors.💯

There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Richez003) [reply style.css:461]: done

File: index.html (hunk 2 above)
Comment (Richez003) [reply index.html:282]: please kindly refresh your Browser, its the same  length from my end thanks

File: index.html (hunk 3 above)
Comment (Richez003) [reply index.html]: done

File: style.css (hunk 4 above)
Comment (Richez003) [reply style.css]: done

File: style.css (hunk 5 above)
Comment (Richez003) [reply style.css]: done

File: style.css (hunk 6 above)
Comment (Richez003) [reply style.css]: done

File: style.css (hunk 7 above)
Comment (Richez003) [reply style.css]: done

File: style.css (hunk 8 above)
Comment (Richez003) [reply style.css]: done

Comment (Richez003) [reply style.css]: done

File: style.css (hunk 7 above)
Comment (Richez003) [reply style.css]: done

File: style.css (hunk 5 above)
Comment (Richez003) [reply style.css]: done

File: style.css (hunk 4 above)
Comment (Richez003) [reply style.css]: done

File: index.html (hunk 3 above)
Comment (Richez003) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (Richez003) [reply index.html:282]: done,

File: style.css (hunk 1 above)
Comment (Richez003) [reply style.css:461]: done

File: index.html (hunk 2 above)
Comment (BenMukebo) [reply index.html:282]: Send me a screenshot I wanna see?

File: index.html [hunk 9]
```diff
@@ -39,33 +44,15 @@ <h2 class="sd">
...
-                        <div class="items">
-                        <ul>
-                            <li>Ruby on rails</li>
-                            <li>Css</li>
-                            <li>Java script</li>
-                            <li>Html</li>
-                        </ul>
-                    </div>
-                    <div class="button-cont">
-                        <button type="button" class="potfolio-but">See Project</button>
```
Comment (BenMukebo) [inline index.html:65]: - [ OPTIONAL] Try to improve your design on this section:
- You can use `text-aline: center` to your `card-lists` container,
- And Add `border-radius` to the language `li`



back

File: index.html [hunk 10]
```diff
@@ -245,35 +269,56 @@ <h1>About Me</h1>
...
-                   
-                    <input type="text" maxlength="30" placeholder="Full Name" required>
-            
-                
-                    <input type="email" name="email" placeholder="Enter address" required>
-            
+                   <div class="form-style">
+                       <div class="input">
+                    <input type="text" maxlength="30" placeholder="First Name" required>
+                    <input type="text" maxlength="30" placeholder="Last Name" required>
```
Comment (BenMukebo) [inline index.html:282]: - [OPTIONAL ] Great job 👍🏻 However, it does not follow the Figma design closely enough, What do you think to add more `padding-top`?

- Your design:

bad3

- figma design:

go111111

File: index.html [hunk 11]
```diff
@@ -245,35 +269,56 @@ <h1>About Me</h1>
...
+                </div> 
+    
+                <div class="flex-container1">
+                        <div><a href="https://twitter.com/richardfriday14"><img src="./images/twitter.png" alt="twitter"></a></div>
+                        <div><a href="https://www.linkedin.com/in/richard-friday-54980718a"><img src="./images/Linkedin.png" alt="linkedin"></a></div>
+                        <div><a><img src="./images/medium.png" alt="medium"></a></div>
+                        <div><a href="https://github.com/Richez003"><img src="./images/github.png" alt="github"></a></div>
+                        <div><a href="https://angel.co/jobs?ref=onboarding"><img src="./images/angellist.png" alt="angellist"></a></div>
+            </div>             
+
```
Comment (BenMukebo) [inline index.html:322]: - [OPTIONAL ] Kindly add this vertical line before your social icons

- Your Design

ba2

- Figma design

go5555

**Review by BenMukebo — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### **Hi @Richez003** , 

### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


### To Highlight :1st_place_medal: 
- No Linter errors :heavy_check_mark: 
- Githubflow was correctly used. 👍🏻 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
- [ OPTIONAL] Please, Write a meaningful title for your pull request and always add a short summary of changes introduced in the pull request. you can check the  [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

- [ OPTIONAL] Kindly you can consider to add a screenshot of your project design in your README.md file in order to make it more professional

Cheers and Happy coding!👏👏👏💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BenMukebo — CHANGES_REQUESTED** [review]
- [OPTIONAL ] Kindly your design is really broken from 768px up to 1400px desktop media query ⬇️, I strongly recommend you fix that since the previous reviewer did not notice that. In meantime, I've approved this project

**Pull request merged**